src/main/java/uk/gov/hmcts/reform/civil/service/citizenui/DashboardClaimInfoService.java

```diff
@@ -92,7 +105,16 @@ private List<DashboardClaimInfo> sortOcmcCases(List<DashboardClaimInfo> ocmcCase
 
     private SearchResult getCCDCasesForDefendant(String authorisation, int startIndex) {
         log.info("-----------calling CCD claims-------------");
-        SearchResult claims = coreCaseDataService.getCCDDataBasedOnIndex(authorisation, startIndex);
+        SearchResult claims =
+            coreCaseDataService.getCCDDataBasedOnIndex(authorisation, startIndex, "data.defendantUserDetails.email");
```

Comment:
I am not sure whether having knowledge of db field in here is a good idea. perhaps it should still stay inside the coreCaseDataService. and you can have two public methods there getCCDClaimsForLipClaimant and getCCDClaimsForLipDefendant?

Reply:
moved the methods to coreCaseDataService with mentioned naming